ts/components/conversation/message/message-content/MessageAttachment.tsx

```diff
@@ -149,52 +158,35 @@ export const MessageAttachment = (props: Props) => {
         />
       </div>
     );
-  } else {
-    const { pending, fileName, fileSize, contentType } = firstAttachment;
-    const extension = getExtensionForDisplay({ contentType, fileName });
-
-    return (
-      <div className={classNames('module-message__generic-attachment')}>
-        {pending ? (
-          <div className="module-message__generic-attachment__spinner-container">
-            <Spinner size="small" />
-          </div>
-        ) : (
-          <div className="module-message__generic-attachment__icon-container">
-            <div
-              role="button"
-              className="module-message__generic-attachment__icon"
-              onClick={onClickOnGenericAttachment}
-            >
-              {extension ? (
-                <div className="module-message__generic-attachment__icon__extension">
-                  {extension}
-                </div>
-              ) : null}
-            </div>
-          </div>
-        )}
-        <div className="module-message__generic-attachment__text">
-          <div
-            className={classNames(
-              'module-message__generic-attachment__file-name',
-              `module-message__generic-attachment__file-name--${direction}`
-            )}
-          >
-            {fileName}
-          </div>
+  }
+  const { pending, fileName, fileSize, contentType } = firstAttachment;
+  const extension = getExtensionForDisplay({ contentType, fileName });
+
+  return (
+    <div className="module-message__generic-attachment">
+      {pending ? (
+        <div className="module-message__generic-attachment__spinner-container">
+          <Spinner size="small" />
+        </div>
+      ) : (
+        <div className="module-message__generic-attachment__icon-container">
           <div
-            className={classNames(
-              'module-message__generic-attachment__file-size',
-              `module-message__generic-attachment__file-size--${direction}`
-            )}
+            role="button"
```

Comment:
do we want to use aria labels?

Reply:
I'll add aria labels when those items are styled components. I don't want to make too many changes until most of those pending PRs are merged

Reply:
Fair enough